tests/stdc/enc/enc.cpp

```diff
@@ -174,6 +174,7 @@ static void _test_time_functions(void)
     const uint64_t SEC_TO_USEC = 1000000UL;
     const uint64_t JAN_1_2018 = 1514786400UL * SEC_TO_USEC;
     const uint64_t JAN_1_2050 = 2524629600UL * SEC_TO_USEC;
+    const uint64_t SLEEP_SECS = 3;
```

Comment:
3 seconds is pretty long, given that we're calling sleep multiple times. Is the test equally effective if we reduce this to 1?

Reply:
Yes, I changed that.